src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/ContestedCourtHelper.java

```diff
@@ -105,6 +113,111 @@ public static String getSelectedCourt(CaseDetails caseDetails) {
         }
     }
 
+    public static String getSelectedInterimHearingFrc(Map<String, Object> interimHearingData) {
```

Comment:
Good candidate to refactor here to use interface and handler pattern ie. getIntermimHearingFRC and canHandle(region)